main.py

```diff
@@ -847,18 +854,43 @@ def escape_appscript(text: str) -> str:
         ")"
     )
 
+    if google_drive_image_link:
+        # Extract image file ID from the passed link
+        file_id_matches = re.match(
+            r"https://drive.google.com/file/d/(.+)/view", google_drive_image_link
+        )
+        if file_id_matches:
+            file_id = file_id_matches.group(1)
+
+            # Add an image to the form
+            appscript += (
+                f';f.addImageItem().setImage(DriveApp.getFileById("{file_id}"))'
+            )
+
     # Add comments/suggestions to form
     appscript += ";f.addParagraphTextItem().setTitle('Comments/suggestions')"
 
+    # Print links to the form
+    appscript += (
+        ';console.log("Edit: "+f.getEditUrl()+"\\n\\nPublished: "+f.getPublishedUrl());'
```

Comment:
I wish these could be clickable links, but as far as I can see you can't get clickable links in the app script execution log.